### ISSUE TITLE
Add "Duration" to succeeded list view as separate column.

### ISSUE THREAD

**Issue by markalanevans** [HangfireIO/Hangfire#797]
Can you add "Duration" as a column next to "Total Duration" in the succeeded view?
It is helpful to see how long the job actually took to run when compared to other jobs, not so much how long it sat in the queue plus time to run.

**Comment by hankir** [comment]
fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff

**Comment by alexanderdevoid** [comment]
Yeah this would be very convenient especially when you have hundreds of homogeneous jobs and you want to take a closer look at those that sink the performance.

You can always rely on stats databases to do this but since the duration value is already known to HF it wouldn't be much of a hassle to implement I suppose :D
